### PR TITLE
build: normalize json with webpack loader

### PR DESCRIPTION
**Pull request by frenautvh**
issue : {0} was not replaced with {{ t0 }} when importing translation json files
fix : add webpack loader

### CODE REVIEW

File: packages/manager/tools/component-rollup-config/src/plugins/common/index.ts [hunk 1]
```diff
@@ -0,0 +1,15 @@
+const translationNormalize = (text) => {
+  if (text) {
+    const filtered = text
+      .toString()
+      .replace(/&#13;\n/g, ' ') // carriage returns
+      .replace(/&#160;/g, ' ') // spaces
+      .replace(/\{(\s?\d\s?)\}/g, '{{t$1}}'); // {0} => {{t0}}
```
Comment (FredericEspiau) [inline packages/manager/tools/component-rollup-config/src/plugins/common/index.ts]:
```suggestion
      .replace(/\{(\s*\d\s*)\}/g, '{{t$1}}'); // {0} => {{t0}}
```

File: packages/manager/tools/component-rollup-config/src/plugins/common/index.ts [hunk 2]
```diff
@@ -0,0 +1,15 @@
+const translationNormalize = (text) => {
+  if (text) {
+    const filtered = text
+      .toString()
+      .replace(/&#13;\n/g, ' ') // carriage returns
+      .replace(/&#160;/g, ' ') // spaces
```
Comment (FredericEspiau) [inline packages/manager/tools/component-rollup-config/src/plugins/common/index.ts]: https://stackoverflow.com/questions/31790440/regex-to-replace-no-break-space < may be more appropriate

File: packages/manager/tools/component-rollup-config/src/plugins/common/index.ts [hunk 3]
```diff
@@ -0,0 +1,15 @@
+const translationNormalize = (text) => {
+  if (text) {
+    const filtered = text
+      .toString()
+      .replace(/&#13;\n/g, ' ') // carriage returns
```
Comment (FredericEspiau) [inline packages/manager/tools/component-rollup-config/src/plugins/common/index.ts]:
```suggestion
      .replace(/\r\n/g, ' ') // carriage returns
```

File: packages/manager/tools/component-rollup-config/src/plugins/common/index.ts (hunk 2 above)
Comment (frenautvh) [reply packages/manager/tools/component-rollup-config/src/plugins/common/index.ts]: it's a regex that replace the html code `&#160;` by a whitespace in our json, it's different from replacing a single no-break-space character

File: packages/manager/tools/component-rollup-config/src/plugins/common/index.ts (hunk 3 above)
Comment (frenautvh) [reply packages/manager/tools/component-rollup-config/src/plugins/common/index.ts]: same here the regex is about replacing the `&#13;` html code

Comment (FredericEspiau) [reply packages/manager/tools/component-rollup-config/src/plugins/common/index.ts]: Why not replacing it why `\r` then ?

**Pull request merged**